src/coreclr/build-runtime.cmd

```diff
@@ -361,10 +361,18 @@ if %__BuildNative% EQU 1 (
     echo %__MsgPrefix%Commencing build of native components for %__TargetOS%.%__TargetArch%.%__BuildType%
 
     REM Set the environment for the native build
-    set __VCTargetArch=amd64
-    if /i "%__HostArch%" == "x86" ( set __VCTargetArch=x86 )
-    if /i "%__HostArch%" == "arm64" (
-        if /i "%__ProcessorArch%" == "ARM64" (set __VCTargetArch=arm64) else (set __VCTargetArch=x86_arm64)
+    if /i "%__ProcessorArch%" == "AMD64" (
```

Comment:
We should either take `PROCESSOR_ARCHITEW6432` into account in both scripts; or in neither script. Does `PROCESSOR_ARCHITEW6432` make a difference - is it set in x64 emulated cmd shell on Arm64 machines?

Reply:
According to [this article](https://blog.differentpla.net/blog/2013/03/10/processor-architew6432/) it's there for x86 compatibility, so presumably it's obsolete for our use case?

Reply:
Given that we now only need to check for ARM64 I think we can just remove this.